Monika After Story/game/zz_windowreacts.rpy

```diff
@@ -0,0 +1,315 @@
+#NOTE: This ONLY works for Windows atm
+
+#Whether Monika can use notifications or not
+default persistent._mas_enable_notifications = False
+
+#Whether Monika can see your active window or not
+default persistent._mas_windowreacts_windowreacts_enabled = False
+
+#Persistent windowreacts db
+default persistent._mas_windowreacts_database = dict()
+
+#A global list of events we DO NOT want to unlock on a new session
+default persistent._mas_windowreacts_no_unlock_list = list()
+
+#A dict of locations where notifs are used, and if they're enabled for said location
+default persistent._mas_windowreacts_notif_filters = dict()
+
+init -10 python in mas_windowreacts:
+    #We need this in case we cannot get access to the libs, so everything can still run
+    can_show_notifs = True
+
+    #The windowreacts db
+    windowreact_db = {}
+
+    #Group list, to populate the menu screen
+    #NOTE: We do this so that we don't have to try to get a notification
+    #In order for it to show up in the menu and in the dict
+    _groups_list = [
+        "Topic Alerts",
+        "Window Reactions",
+    ]
+
+init python:
+    import os
+    #The initial setup
+
+    #We can only do this on windows
+    if renpy.windows:
+        #We need to extend the sys path to see our packages
+        import sys
+        sys.path.append(renpy.config.gamedir + '\\python-packages\\')
+
+        #We try/catch/except to make sure the game can run if load fails here
+        try:
+            #Going to import win32gui for use in destroying notifs
+            import win32gui
+            #Import win32api so we know if we can or cannot use notifs
+            import win32api
+
+        except ImportError:
+            #If we fail to import, then we're going to have to make sure nothing can run.
+            store.mas_windowreacts.can_show_notifs = False
+
+            #Log this
+            store.mas_utils.writelog(
+                "[WARNING]: win32api/win32gui failed to be imported, disabling notifications.\n"
+            )
+
+        else:
```

Comment:
I assume this else is supposed to match with `if.renpy.windows`? indentation doesnt match

Reply:
Else is for the try/catch/else block. We only import balloontip and such if it's possible. Otherwise we don't.

Reply:
interesting, have not seen this used in practice, think its worth commenting that its for the try-except so its not assumed to be part of the if.

Reply:
Sure